Automattic/jetpack
# mirrors: Use a token secret in the autotagger

#### Changes proposed in this Pull Request:

We're going to want to trigger some other actions on tag, but taggings
with the default `GITHUB_TOKEN` specifically do not trigger other actions.

So we'll need to add a token secret to all the mirrors using autotagger.
Added a tool to do mass updates of repo secrets to help with that.

#### Jetpack product discussion


p9dueE-3eM-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Try out the new tool (on a personal repo, preferably).
* Merge this and #20455, then release eslint-changed. See that the autotagging happens and triggers the npm publish.
